From 89948e58fd0f80def0912c18c503912873aa9c48 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 9 Jul 2015 16:14:50 +0200
Subject: [PATCH] crypt: install drbg unconditionally in hostonly mode

older kernels had the drbg kernel module and didn't need it
---
 modules.d/90crypt/module-setup.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/90crypt/module-setup.sh b/modules.d/90crypt/module-setup.sh
index dbc87a43..bd37d9a5 100755
--- a/modules.d/90crypt/module-setup.sh
+++ b/modules.d/90crypt/module-setup.sh
@@ -23,7 +23,8 @@ depends() {
 }
 
 installkernel() {
-    instmods dm_crypt =crypto drbg
+    instmods dm_crypt =crypto
+    hostonly="" instmods drbg
 }
 
 cmdline() {
